kaiax/builder/impl/tx_pool_wrapping.go

```diff
@@ -60,6 +60,11 @@ func (b *BuilderWrappingModule) PreAddTx(tx *types.Transaction, local bool) erro
 	if knownTx, ok := b.knownTxs.get(tx.Hash()); ok && knownTx.elapsedTime() < KnownTxTimeout {
 		return ErrUnableToAddKnownBundleTx
 	}
+
+	if uint(len(b.knownTxs)) >= b.txBundlingModule.GetMaxBundleTxsInPool() {
```

Comment:
knownTxs isn't queue. to be accurate, you need to use the number of knwonTxs without demoted one.